ubi8-openjdk-11.yaml

```diff
@@ -19,9 +19,11 @@ labels:
 - name: "com.redhat.component"
   value: "openjdk-11-ubi8-container"
 - name: "usage"
-  value: "https://access.redhat.com/documentation/en-us/openjdk/11/html/using_openjdk_11_source-to-image_for_openshift/index"
+  value: &docs "https://access.redhat.com/documentation/en-us/openjdk/11/html/using_source-to-image_for_openshift_with_red_hat_build_of_openjdk_11/index"
```

Comment:
I'd rather keep this consistent with other images. I.e. point to https://jboss-container-images.github.io/openjdk/ instead. It's up-to-date and in-sync with the releases. The product doc could gain a reference link to the upstream doc as well.

Reply:
I'd be happy doing that. And also linking to the product docs from the upstream docs (we link from the README already).

Reply:
I agree with @jerboaa . I would like to see consistency across the documentation.

Reply:
OK it's decided; I'll push a subsequent commit that unifies them all with the upstream/community URI. Thank you for the feedback!